docroot/modules/custom/foia_webform/src/FoiaSubmissionQueueingService.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Drupal\foia_webform;
+
+use Drupal\foia_request\Entity\FoiaRequestInterface;
+
+/**
+ * Class FoiaSubmissionQueueingService.
+ */
+class FoiaSubmissionQueueingService implements FoiaSubmissionQueueingServiceInterface {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function addRequestToQueue(FoiaRequestInterface $foiaRequest) {
+    /** @var \Drupal\Core\Queue\QueueFactory $queueFactory */
+    $queueFactory = \Drupal::service('queue');
```

Comment:
We should probably make this class an actual service and inject these items using dependency injection.